### PR TITLE
Ers 1694/core metrics

### PR DESCRIPTION
**Pull request by dpipemazo**
## Change description

Updates build, documentation, containers to provide a basic system-level dashboard with Atom where we're utilizing metrics to monitor how a host is doing

Dashboards that can be produced by default with this:
dashboards1
dashboards2
dashboards3
dashboards4

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: metrics/monitoring/metrics_monitor.py [hunk 1]
```diff
@@ -642,204 +609,12 @@ def sensors_metrics_update_item(item, sensor, val):
...
-                        timedelta = (
-                            cpu_times_time[proc] - cpu_times_time_last[proc]
-                        ) / 100.0
-                        output[proc][thread] = (
-                            curr[proc][thread] - prev[proc][thread]
-                        ) / timedelta
-
-        return output
-
-    # All of the metrics we're tracking on a per-process basis
```
Comment (aywaldron) [inline metrics/monitoring/metrics_monitor.py:737]: Were these process metrics not really useful?

**Review by aywaldron — COMMENTED** [review]
This looks great, thanks for the preview. Are we always going to run a monitor service on all edge stacks going forward?

If its not too much work, it'd be cool to split the dashboards into sections, e.g. "Network" section, that you can open or minimize.

File: metrics/monitoring/metrics_monitor.py (hunk 1 above)
Comment (aywaldron) [reply metrics/monitoring/metrics_monitor.py:737]: NVM the above, answered in scrum.

**Comment by dpipemazo** [comment]
> Are we always going to run a monitor service on all edge stacks going forward?

Yes, I'd like to. We can turn it off if need be. 

> it'd be cool to split the dashboards into sections, e.g. "Network", "CPU", that you can open or minimize.

Ah, yeah this would be awesome! Is this something easy enough to do in Grafana? I haven't explored too much out of line chart setup and aspire to get to your teams' dashboard level (gauges, etc.) someday :smile:

**Comment by dpipemazo** [comment]
New home dashboard along with this contains better linking to the system dashboard + some of the more critical graphs from the system dashboard. 

home_dashboard


> it'd be cool to split the dashboards into sections, e.g. "Network", "CPU", that you can open or minimize.

I'll follow a follow-on ticket in the improvements for this to come back to it; given time constraints I'd like to ship this as-is for the next release.

**Pull request merged**